UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/SubscriptionServiceJob.cs

```diff
@@ -35,6 +44,11 @@ public async Task ProcessQueueMessage([QueueTrigger("%SubscriptionStorageConfigu
             SubscriptionRequestResult subscriptionRequestResult = new(subscriptionMessage);
             if (subscriptionMessage.IsActive)
             {
+                ExternalNotificationEntity externalNotificationEntity = new()
+                {
+                    ResponseStatusCode = _d365CallbackConfiguration.Value.SucceededStatusCode,
+                    ResponseDetails = Convert.ToString(DateTime.UtcNow)
+                };
                 try
                 {
                     eventSubscription = await _subscriptionServiceData.CreateOrUpdateSubscription(subscriptionMessage, CancellationToken.None);
```

Comment:
Is `eventSubscription` goiung to be used in a future PBI?  It is currently assigned but never used.

Reply:
Yes, considering it will be used in future, It is assigned. If anytime its not used, we will remove at that moment